acoular/tbeamform.py

```diff
@@ -228,7 +228,7 @@ def _get_rm(self):
     numchannels = Delegate('grid', 'size')
 
     #: Spatial weighting function.
-    weights = Trait('none', possible_weights, desc='spatial weighting function')
+    weights = Map(possible_weights, default_value='none', desc='spatial weighting function')
```

Comment:
could you explain this line?

Reply:
The weights trait maps a string to a function. `possible weights` is an unordered dictionary and has therefore no defined first entry. Therefore the 'none' key needs to be defined as default.
It could well be that this design is not ideal, but this PR is not about changing this.